common/djangoapps/student/reports.py

```diff
@@ -0,0 +1,69 @@
+import unicodecsv as csv
+
+from django.utils import timezone
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
+
+from lms.djangoapps.grades.models import PersistentCourseGrade
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def csv_course_report(f):
+    field_names = [
+        'Candidate RSA ID number',
+        'Candidate First Name',
+        'Candidate Last Name',
+        'Enrollment Date',
+        '% Complete to date',
+        'Completion Date',
+        'Duration between enrollment and completion',
+    ]
+    writer = csv.DictWriter(f, fieldnames=field_names)
+    for course_overview in CourseOverview.objects.all():
+        course_title = [course_overview.id, course_overview.display_name]
+        course_title.extend([''] * (len(field_names) - 2))
+        writer.writerow(dict(zip(field_names, course_title)))
+        writer.writeheader()
+
+        for enrollment in course_overview.courseenrollment_set.filter(is_active=True):
+            row_values = []
+
+            user = enrollment.user
+            row_values.append(user.username)
+            row_values.append(user.first_name)
+            row_values.append(user.last_name)
+
+            row_values.append(timezone.localtime(enrollment.created))
```

Comment:
what localtime has been set on the server?  It needs to be set to GMT +2

Reply:
Also note that enrollment date <> course start date. Someone can be enrolled months before they actually start the course